cime_config/machines/config_compilers.xml

```diff
@@ -1531,6 +1531,22 @@ flags should be captured within MPAS CMake files.
   </SLIBS>
 </compiler>
 
+<compiler COMPILER="gnu" MACH="lobata">
+  <SUPPORTS_CXX>TRUE</SUPPORTS_CXX>
+  <CPPDEFS>
+    <append MODEL="gptl"> -DHAVE_VPRINTF -DHAVE_GETTIMEOFDAY -DHAVE_BACKTRACE </append>
+  </CPPDEFS>
+  <FFLAGS>
+    <append DEBUG="FALSE">-fno-unsafe-math-optimizations</append>
+    <append DEBUG="TRUE"> -g -fbacktrace -fbounds-check -ffpe-trap=invalid,zero,overflow</append>
+  </FFLAGS>
+  <SLIBS>
+    <append> -L$ENV{NETCDF_FORTRAN_PATH}/lib/ -lnetcdff -L$ENV{NETCDF_C_PATH}/lib/ -lnetcdf -lcurl -llapack -lblas </append>
+  </SLIBS>
+  <SCC MPILIB="mpi-serial">gcc</SCC>
+  <SFC MPILIB="mpi-serial">gfortran</SFC>
+</compiler>
+
```

Comment:
`config_compilers.xml` has been completely replaced by `cmake_macros/*`. This is not necessary.

Reply:
I had assumed this might be good to include if a user is still using an older version of e3sm/cime and was the reason for the file still being included on master.